pkgs/development/python-modules/maxminddb/default.nix

```diff
@@ -10,13 +12,18 @@ buildPythonPackage rec {
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "f4d28823d9ca23323d113dc7af8db2087aa4f657fafc64ff8f7a8afda871425b";
+    sha256 = "0ns2f6lgv2ksizzn9z7sazva8yh8na6szirx24yk48yav4iqilpl";
```

Comment:
FYI, this is the same hash:

```
# nix to-base32 sha256:f4d28823d9ca23323d113dc7af8db2087aa4f657fafc64ff8f7a8afda871425b
0ns2f6lgv2ksizzn9z7sazva8yh8na6szirx24yk48yav4iqilpl
```

No real need to change it. The reason why base16 is used for python packages is because you can see it on pypi.org: https://pypi.org/project/maxminddb/#copy-hash-modal-35eac021-2bec-4018-aa04-d0e1050eaa09